src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/SecurityUtils.cs

```diff
@@ -46,6 +46,8 @@ public static DateTime MinUtcDateTime
             }
         }
 
+        public static bool RequiresFipsCompliance { get; internal set; }
+
```

Comment:
Fips compliance requirements is only a NetFx thing. Just use the factory methods for the various security classes (e.g. SHA256.Create()) and we don't need to worry about Fips compliance.